apps/rush-lib/src/api/ExperimentsConfiguration.ts

```diff
@@ -44,10 +44,8 @@ export interface IExperimentsJson {
   /**
    * If true, the multi-phase commands feature is enabled. To use this feature, create a "phased" command
    * in common/config/rush/command-line.json.
-   *
-   * THIS FEATURE IS NOT READY FOR USAGE YET. SEE GITHUB #2300 FOR STATUS.
    */
-  _multiPhaseCommands?: boolean;
+  multiPhaseCommands?: boolean;
```

Comment:
The design note #2300 seems to have converged on calling this feature "phased commands".  If so, we should update the experiment to reflect that, and then search+replace "multiphase" and "multi-phase" everywhere to make the terminology consistent.